graspy/align/seedless_procrustes.py

```diff
@@ -0,0 +1,373 @@
+# Copyright 2020 NeuroData (http://neurodata.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import ot
+import numpy as np
+from sklearn.utils import check_array
+
+from .base import BaseAlign
+from .sign_flips import SignFlips
+from .orthogonal_procrustes import OrthogonalProcrustes
+
+
+class SeedlessProcrustes(BaseAlign):
+    """
+    Implements an algorithm that matches two datasets using an orthogonal
+    matrix. Unlike OrthogonalProcrustes, this does not use a matching between
+    entries. It can even be used in the settings when the two datasets do not
+    have the same number of vertices.
```

Comment:
IMO docs for this shouldn't only talk about vertices, as could be used on arbitrary points

Reply:
sure. i don't think we have an applications outside of graph embeddings yet, but i don't mind saying it generally. especially since i did so in orthogonal one; i prob meant to say "entries" anyway.

Reply:
changed to entries.

Reply:
also added a comment regarding using in graph setting.